docs/core_tenets.md

```diff
@@ -0,0 +1,18 @@
+## Jujutsu's Core Tenets
+
+Jujutsu's core tenets are:
+
+ * User-friendliness: Making the  working copy a commit is simpler. This is
+ how the project started.
+ * The "repository", so the commit graph is the source of truth. The working
```

Comment:
"The repository is the source of truth: Most commands should operate of the commit graph. The working copy is just one way of editing commits."

Reply:
done